htu21d.py

```diff
@@ -6,24 +6,25 @@ class HTU21D(object):
     ISSUE_TEMP_ADDRESS = 0xE3
     ISSUE_HU_ADDRESS = 0xE5
 
-    def __init__(self, scl, sda):
+    def __init__(self, scl=None, sda=None, i2c=None):
         """Initiate the HUT21D
-
         Args:
             scl (int): Pin id where the sdl pin is connected to
             sda (int): Pin id where the sda pin is connected to
         """
-        self.i2c = I2C(scl=Pin(scl), sda=Pin(sda), freq=100000)
+        if i2c is not None:
+            self.i2c=i2c
+        else:
+            self.i2c = I2C(scl=Pin(scl), sda=Pin(sda), freq=100000)
+        self.running=False
```

Comment:
For what reason is the running member variable here?

Reply:
To be precise, it is of no importance here and can indeed be removed again.
This is a leftover of my port to async.